reconcile/external_resources/aws.py

```diff
@@ -188,11 +189,14 @@ def _get_timeout_minutes(
         self,
         timeout: str,
     ) -> int:
-        if not re.match(AWSRdsFactory.TIMEOUT_RE, timeout):
+        if not (match := re.fullmatch(AWSRdsFactory.TIMEOUT_RE, timeout)):
             raise ValueError(
-                f"Invalid RDS instance timeout format: {timeout}. Specify timeout in minutes(m)."
+                f"Invalid RDS instance timeout format: {timeout}. Specify a duration using 'h' and 'm' only."
```

Comment:
Please add an example:

```suggestion
                f"Invalid RDS instance timeout format: {timeout}. Specify a duration using 'h' and 'm' only. E.g. 2h30m"
```

Reply:
added